lazybytez/jojo-bot-website
# Adds less and prettier tailwind class sorter

## Description

Adds less suppoert and tailwind class sorter.

## How can this be tested?

Start dev instance and see if it works.  
Additionally, create multiple CSS files and import them into app.less to see if imports are working correctly

## Additional context

Execute ```yarn format``` or ```yarn run format``` to run prettier and ```yarn lint``` or ```yarn run lint``` to check if the code styles are correct.  
If you don't want to execute the formatter every single time, consider installing the prettier addon, setting prettier as your default code formatter and enable formatting on paste and save!

![grafik](https://user-images.githubusercontent.com/42189915/213927141-9cb53f45-d9d7-45cf-8e03-f6adf2245e33.png)
